internal/cmd/schema-registry/command_config_describe.go

```diff
@@ -18,6 +19,11 @@ import (
 
 type configOut struct {
 	CompatibilityLevel string `human:"Compatibility Level" serialized:"compatibility_level"`
+	CompatibilityGroup string `human:"Compatibility Group" serialized:"compatibility_group"`
+	DefaultMetadata    string `human:"Default Metadata" serialized:"default_metadata"`
+	OverrideMetadata   string `human:"Override Metadata" serialized:"override_metadata"`
```

Comment:
Do you mean "overridden"?

Reply:
The naming is following https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/Config.java#L40-L45